lms/djangoapps/instructor/static/instructor/ProblemBrowser/components/Main/Main.jsx

```diff
@@ -27,8 +27,20 @@ export default class Main extends React.Component {
 
     return (
       <div className="problem-browser">
-        <Button onClick={this.handleToggleDropdown} label={gettext('Select a section or problem')} />
-        <input type="text" name="problem-location" value={selectedBlock} disabled />
+        {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
+        <span
+          onClick={this.handleToggleDropdown}
+          className={['problem-selector']}
+        >
+          <span>{selectedBlock || 'Select a section or problem'}</span>
+          <span style={{ float: 'right' }}>
```

Comment:
if you try to use class `pull-right` you won't need custom styles here.

Reply:
Ok, sure I will try this and update code accordingly.